samples/RestoreSQLDB/Restore-CohesityRemoteMSSQLObject-Wrapper.ps1

```diff
@@ -0,0 +1,148 @@
+<#
+    Use the generic cmdlet to create an MSSQL remote restore task
+
+    Example usage:
+        Restore-CohesityRemoteMSSQLObject-Wrapper.ps1 -JobId 1234 -SqlHost x.x.x.x -SqlObjectName "MSSQLSERVER/database" -TargetHost y.y.y.y -CaptureTailLogs:$false -NewDatabaseName database_new -NewInstanceName SQL2016 -TargetDataFilesDirectory "C:\" -TargetLogFilesDirectory "C:\temp" -DbRestoreOverwritePolicy:$true
```

Comment:
Can you write multiple examples to show restore from current vs remote cluster and stuff ?